committees/project-director-election-process.md

```diff
@@ -0,0 +1,34 @@
+# Leadership Council _Project Director Election Process_ Committee
+
+## Goals
+
+* Design the election process for the next set of Rust Foundation project directors (ratified by the leadership council)
+* Clearly document and communicate the process and support its execution
+* Track, review, and document the effectiveness of the process
+
+## Delegated Responsibilities
+
+* None, this committee exists to create a proposal which will be consented to by the council
+
+## Duration
+
+* Temporary. This committee may be recreated each time we do a project director election in order to help support the process.
```

Comment:
I'm perhaps not understanding the differences between "recreated" and "new committee...to potentially form". Those seem the same to me. Can you clarify how those seem different?  Is the key difference that it would re-evaluate the goals and responsibilities each time?

My understanding by *temporary* is that it exists until it has fulfilled its goals, which I'm not sure we can put an exact time frame on. Putting some expected rough estimates on how long it might take could be helpful, but I also imagine quite difficult to estimate accurately.  One suggestion is to specify that it should complete the first half of proposing a process before the terms are up (is that roughly 6 weeks?), and with enough time for the Council to approve it, and then a rough estimate for how long to spend reviewing the effectiveness afterward.  Does that sound about right?

Reply:
I tried resolving this one, but I'm not confident in my changes, so I'm leaving this open. Please mark this resolved if you're happy with my changes.

Reply:
@ehuss my thought is: there is a difference between *forming* (proposing) a process and *executing* a process. It doesn't really seem like this committee would be involved in executing the process, only forming it. I would expect that unless there is some fundamental breakdown in the process this time around, we shouldn't need to form it again. (And even then, I imagine that may be covered by the post-selection evaluation).

Given that, if a new committee would be formed, I would expect its role would be to help facilitate the selection process, rather than come up with a new process (as this committee has)

Reply:
This is fine by me now.